File: Java/Iniciante/1008 - Salário..java
/*
* Autor: Pierre Vieira
* Data de submissão: 19/01/2020 17:15:54
*/
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner teclado = new Scanner(System.in);
        int a, b;
        double c;
        a = teclado.nextInt();
        b = teclado.nextInt();
        c = teclado.nextDouble();
        System.out.println("NUMBER = "+a);
        System.out.printf("SALARY = U$ %.2f\n", b*c);
    }
}

File: C#/Iniciante/1008 - Salário.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 13/05/2020 12:28:28
*/
using System;
using System.Globalization;

namespace URI
{
    class URI
    {
        static void Main(string[] args)
        {
            int numero_funcionario, qtde_horas_trabalhads;
            double valor_recebido_por_hora_trabalhada;
            numero_funcionario = int.Parse(Console.ReadLine());
            qtde_horas_trabalhads = int.Parse(Console.ReadLine());
            valor_recebido_por_hora_trabalhada = double.Parse(Console.ReadLine());
            Console.WriteLine($"NUMBER = {numero_funcionario}\nSALARY = U$ {(qtde_horas_trabalhads*valor_recebido_por_hora_trabalhada).ToString("F2", CultureInfo.InvariantCulture)}");
        }
    }
}
